addons/web/static/src/views/list/list_renderer.js

```diff
@@ -244,6 +248,10 @@ export class ListRenderer extends Component {
         });
         this.isRTL = localization.direction === "rtl";
         this.uniqueRendererClass = uniqueId("o_list_renderer_");
+        this.dialogClose = [];
+        onWillDestroy(() => {
+            this.dialogClose.forEach((close) => close());
```

Comment:
As far as I understand this part does nothing. There's no `dialogClose` concept (as you introduce it), so nobody pushes callbacks in it. So you always iterate on an empty array here. What am I missing?